.github/workflows/testingfarm.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Testing farm tests
+
+on:
+  pull_request_target:
```

Comment:
Any reason to use `pull_request_target` over `pull_request` ? Afaik the only difference is that `pull_request_target` gets a token which can write to the repository, but I'm not sure if that's needed here.

Reply:
Thanks - maybe I should spell this out better. My understanding is that the addition difference is that pull_request_target gets access to the `secrets.TF_API_KEY` across forks but that the regular `pull_request` does not (as a security measure). But maybe I'm missing something here?

Reply:
Yes, afaik that's correct. It's a way to work around the security restrictions of GH Actions and forks.

There's some precedence to outside contributors not being able to trigger CI. We have a "friends of osbuild" group that we add contributors to that we trust and they can trigger CI. Everyone else has to wait for a team member to push the button.